docs/docs-rules/user-stories.md

```
@@ -158,13 +243,69 @@ docs/user-stories/
+
+#### 重要: クラス図との違い
+
+このダイアグラムは**タスク依存ネットワーク図**であり、**UMLクラス図やアーキテクチャ依存図ではない**。
+
+| 観点 | タスク依存ネットワーク図（本図） | クラス依存図 |
+|------|-------------------------------|-------------|
+| 矢印の意味 | 「このタスク完了後、次のタスクを開始できる」 | 「このクラスは別のクラスに依存する」 |
+| 矢印の例 | `Interactor実装 --> Controller実装` | `Controller --> Interactor` |
+| 解釈 | Interactor実装が完了してからController実装を開始 | ControllerがInteractorを参照する |
+
+**例: Clean Architectureにおける矢印の方向**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)